### PR TITLE
Fix output of 'kubectl rollout status'

### PR DESCRIPTION
**Pull request by oomichi**
If running `kubectl rollout status` command, the output is like:
```
  deployment "nginx-deployment" successfully rolled out
```

The corresponding kubectl code also shows it according to https://github.com/kubernetes/kubectl/blob/e95e378e5972064b177a8e71eac2803f55c8c5df/pkg/polymorphichelpers/rollout_status.go#L89

### CODE REVIEW

**Comment by bells17** [comment]
メモ
既存ページの修正なのでベースブランチを変更していただく必要がある気がするので明日確認する

https://kubernetes.io/ja/docs/contribute/localization/#%E6%97%A2%E5%AD%98%E3%81%AE%E3%83%9A%E3%83%BC%E3%82%B8%E3%81%AE%E8%AA%A4%E5%AD%97%E8%84%B1%E5%AD%97%E3%82%84%E5%8F%A4%E3%81%84%E8%A8%98%E8%BF%B0%E3%82%92%E4%BF%AE%E6%AD%A3%E3%81%99%E3%82%8B%E5%A0%B4%E5%90%88%E3%81%AE%E6%89%8B%E9%A0%86

**Comment by oomichi** [comment]
I need to know how to fix this issue and asked on slack: https://kubernetes.slack.com/archives/C1J0BPD2M/p1602195360107400

**Comment by bradtopol** [comment]
Hi @oomichi Thank you for your contribution. You can just fix the content/en version and the localization teams will translate for other languages if they deem your change to be a priority.  Also, each language update should get its own PR.  If you are fluent in a different language and would like to update that content please submit it as a separate PR.
/assign

**Comment by bradtopol** [comment]
Hi @oomichi I ran Kubernetes to verify that your fix of the output is correct.  However I need you to split this PR into two PRs. One for content/en and one for content/ja.  Once you have done that I will approve the one for content/en and will ask a member of the ja localization team to review the ja PR.   Thanks again for your contribution!

File: content/ja/docs/concepts/workloads/controllers/deployment.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ Deploymentによって作成されたReplicaSetを管理しないでください
   コマンドの実行結果は以下のとおりです。
   ```shell
   Waiting for rollout to finish: 2 out of 3 new replicas have been updated...
-  deployment.apps/nginx-deployment successfully rolled out
+  deployment "nginx-deployment" successfully rolled out
```
Comment (bradtopol) [inline content/ja/docs/concepts/workloads/controllers/deployment.md:95]: See my more detailed comment.  the changes to the content/ja file should be placed in its own PR.

**Review by bradtopol — COMMENTED** [review]
I verified the fix of the output is correct but this needs to be split into two PRs, one for each language

**Comment by oomichi** [comment]
@bradtopol Thanks for your response and review on this PR and the slack :-)
OK, let me try it.

/hold

**Comment by bradtopol** [comment]
Great job!!!
/lgtm
/approve
